src/core/js/views/questionView.js

```diff
@@ -279,6 +282,38 @@ define([
         // be a full reset
         resetQuestion: function() {},
 
+        determineButtonState: function() {
```

Comment:
you don't have to redo the if statement if you can't see any way to make it not nested. i couldn't see any way to make it non-nested.